.github/goreleaser.yaml

```diff
@@ -698,9 +698,6 @@ release:
 
     https://github.com/orgs/gnolang/packages?repo_name={{ .ProjectName }}
 
-# Only valid for nightly build
```

Comment:
Wait. Why?

Nighties are currently used.
I have the error on the schema too, because the yaml is referencing the goreleaser non-PRO schema (actually in VS code).
The PRO schema include it and we successfully use a Gorelease Pro version here